controllers/endpoints.js

```diff
@@ -14,7 +14,7 @@ function formatDate(dateString) {
     return `${month}/${day}/${year}, ${hours}:${minutes}:${seconds}`;
 }
 
-async function getEndpoints(req, res) {
+async function getSingleEndpoint(req, res) {
```

Comment:
Just written my own versions using an API call for single host by hostID, check it out it might match your needs.

Reply:
in my last PR for rmNode

Reply:
I think you got confused.
This is a controller function to return an EJS file for single endpoint only, now fetching 1 endpoint data from fleet

Reply:
Ok i understand what you suggested. But i can't use it before you merge it to main :)

Reply:
Go on and merge then, we'll change it in the future don't let it stop the feature